data/json/items/tool_armor.json

```diff
@@ -1961,10 +1961,10 @@
             "fun_bonus": 2,
             "description_frequency": 20,
             "descriptions": [
-                "Your pipes drone loudly as you strike a battle hymn from the chanter.",
-                "Your pipes drone a slow and piercing dirge.",
-                "You strike a lively quick step march on your pipes."
-            ]
+              "pipes drone loudly as it strikes a battle hymn from the chanter.",
+                "pipes drone a slow and piercing dirge.",
+                "strike a lively quick step march on their pipes."
+          ]
```

Comment:
You have tabs in your files. Those must be replaced by spaces.
Notepad++ has tabs to spaces function. It's safe to run this function on entire files because all our jsons are guaranteed not to have tabs.